azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/service/sdk/ChatSDKWrapper.kt

```diff
@@ -68,9 +70,13 @@ internal class ChatSDKWrapper(
         MutableStateFlow(ChatStatus.NONE)
     private val messagesSharedFlow: MutableSharedFlow<MessagesPageModel> =
         MutableSharedFlow()
+    private val chatEventModelSharedFlow: MutableSharedFlow<ChatEventModel> =
+        MutableSharedFlow()
 
     override fun getChatStatusStateFlow(): StateFlow<ChatStatus> = chatStatusStateFlow
     override fun getMessagesPageSharedFlow(): SharedFlow<MessagesPageModel> = messagesSharedFlow
+    override fun getChatEventSharedFlow(): SharedFlow<ChatEventModel> =
```

Comment:
Prefer using val/get() syntax for these.

val chatEventSharedFlow get() = chatEventModelSharedFlow

Reply:
can we typecast while using get() easiely?